planning_interface/move_group_interface/src/wrap_python_move_group.cpp

```diff
@@ -389,8 +392,42 @@ class MoveGroupWrapper : protected py_bindings_tools::ROScppInitializer,
      return constraints_str;
   }
 
+  std::string retimeTrajectory(const std::string& ref_state_str,
```

Comment:
I know it's a departure from the C++ class name, but "IterativeParabolicTimeParameterization" is a crazy long name to inflict on what is supposed to be an easy-to-use interface.  Also, "parameterizeTrajectory" doesn't actually say what it does.  I thought "retimeTrajectory" was a fairly concise description of what the function does.